### PR TITLE
add rokka.sourceimages.deleteWithBinaryHash and deletePrevious on rok…

### PR DESCRIPTION
**Pull request by chregu**
…ka.sourceimages.setSubjectArea

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,12 +215,22 @@ rokka.sourceimages.delete('myorg', 'c421f4e8cefe0fd3aab22832f51e85bacda0a47a')
...
+  .catch(function(err) {});
+```
+
 ### Dynamic metadata
 
 See [the dynamic metadata documentation](https://rokka.io/documentation/references/dynamic-metadata.html) for
 more information.
 
-#### rokka.sourceimages.setSubjectArea(organization, hash, coords) → Promise
+#### rokka.sourceimages.setSubjectArea()(organization, hash, coords, options) → Promise
```
Comment (pstadler) [inline README.md]: Seems like you broke something here.

Comment (chregu) [inline README.md]: I know, checking with michael how to fix it

**Comment by mweibel** [comment]
@chregu can you rebase on master?

**Comment by chregu** [comment]
@mweibel done

File: src/apis/sourceimages.js [hunk 2]
```diff
@@ -203,6 +203,26 @@ export default (state) => {
...
+   *   .then(function(result) {})
+   *   .catch(function(err) {});
+   * ```
+   *
+   * @authenticated
+   * @param  {string}  organization name
+   * @param  {string}  binaryHash   binary image hash
+   * @return {Promise}
+   */
+  sourceimages.deleteWithBinaryHash = (organization, binaryHash) => {
```
Comment (mweibel) [inline src/apis/sourceimages.js]: nitpicking here, but can you name it `deleteByBinaryHash`? Somehow prefer that and also the docblock describes it as `Delete source images **by** its binary hash.`

Comment (mweibel) [inline src/apis/sourceimages.js]: nevermind, other function is also called with `with` so let's stay consistent.

**Pull request merged**